public_suffix_list.dat

```diff
@@ -4150,6 +4153,7 @@ in.na
 cc.na
```

Comment:
cc.na can be removed

Reply:
https://regery.com/en/domains/search/randomeststring.gov.na still lists the following under Namibia:
> [alt.na ](https://regery.com/en/domains/zone/alt.na)[cc.na ](https://regery.com/en/domains/zone/cc.na)[co.na ](https://regery.com/en/domains/zone/co.na)[com.na ](https://regery.com/en/domains/zone/com.na)[edu.na ](https://regery.com/en/domains/zone/edu.na)[info.na ](https://regery.com/en/domains/zone/info.na)[na ](https://regery.com/en/domains/zone/na)[net.na ](https://regery.com/en/domains/zone/net.na)[org.na ](https://regery.com/en/domains/zone/org.na)school.na

Not sure what that means. Should we move e.g. cc.na to the private section instead?